examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,99 @@
+"""Fitting ODNP Datasets for Ksigma
+===================================
+The T1(p) and E(p) integrals are pulled from an H5 file and are fit to extract the cross relaxivity of the sample. The R1(p) is first interpolated by fitting the self relaxivity to a polynomial. This fit is then used as a parameter for fitting the linear regime of the enhancement data that is extrapolated to the highest power. The cross relaxivity is then calculated using the output of these fits as well as the sample parameters that are fed to it (e.g., ppt value, and concentration).
+"""
+from pyspecdata import *
+from sympy import symbols, Symbol, latex
+from scipy.io import loadmat
+
+target_directory = os.path.normpath(getDATADIR("AG_processed_data"))
+h5_folder = "ras.h5"
+nodename = "220616_E37"
+# {{{Dataset Parameters
+SL_conc_M = 455.2223e-6
+ppt = 1.5167e-3
+Ep_pts = 18
```

Comment:
this is a processing file -- why does this need to be set.

Reply:
this is processing the table of integrals for a dataset - Not the raw acquired dataset. The only thing saved to the h5 file here is the table of integrals for Ep and T1

Reply:
Why can't you use ndshape to figure out how many points there are?

Reply:
ndshape returns `[('power',19)]` so instead I set it to `len(Ep.getaxis('power'))`

Reply:
???